plugins/env/app/models/environment_variable_group.rb

```diff
@@ -0,0 +1,18 @@
+class EnvironmentVariableGroup < ActiveRecord::Base
+  default_scope -> { order(:name) }
+
+  has_many :environment_variables
+  has_many :stage_environment_variable_groups
+  has_many :stages, through: :stage_environment_variable_groups
```

Comment:
You don't have to specify the through table if you use the HABTM relationship. See DeployGroups to Stages
```
  has_and_belongs_to_many :environment_variable_groups
```